Cmdline/CKAN-cmdline.csproj

```diff
@@ -54,34 +41,6 @@
     <Compile Include="..\GlobalAssemblyInfo.cs">
       <Link>Properties\GlobalAssemblyInfo.cs</Link>
     </Compile>
-    <Compile Include="Action\AuthToken.cs" />
-    <Compile Include="Action\Available.cs" />
-    <Compile Include="Action\Cache.cs" />
-    <Compile Include="Action\Compare.cs" />
-    <Compile Include="Action\Compat.cs" />
-    <Compile Include="Action\Filter.cs" />
-    <Compile Include="Action\ICommand.cs" />
-    <Compile Include="Action\Import.cs" />
-    <Compile Include="Action\Install.cs" />
-    <Compile Include="Action\ISubCommand.cs" />
-    <Compile Include="Action\GameInstance.cs" />
-    <Compile Include="Action\List.cs" />
-    <Compile Include="Action\Mark.cs" />
-    <Compile Include="Action\Prompt.cs" />
-    <Compile Include="Action\Remove.cs" />
-    <Compile Include="Action\Repair.cs" />
-    <Compile Include="Action\Replace.cs" />
-    <Compile Include="Action\Repo.cs" />
-    <Compile Include="Action\Search.cs" />
-    <Compile Include="Action\Show.cs" />
-    <Compile Include="Action\Update.cs" />
-    <Compile Include="Action\Upgrade.cs" />
-    <Compile Include="ConsoleUser.cs" />
-    <Compile Include="Exit.cs" />
-    <Compile Include="Main.cs" />
-    <Compile Include="Options.cs" />
-    <Compile Include="ProgressReporter.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
```

Comment:
Assume the optimisations make manually defining these not necessary?

Reply:
There are two main styles of `.csproj` file; the older one requires specifying all the files, and the newer one that scans whatever files are in the directory instead. We're switching from the old style to new style, as was previously done for Core, since it's much more convenient for maintenance.

Reply:
I think one of these lines pulls in the standard rules for finding all the files:

https://github.com/KSP-CKAN/CKAN/blob/1b1d181ec9d92f952834e571f5be3db6cb4ba0d4/AutoUpdate/CKAN-autoupdate.csproj#L7
https://github.com/KSP-CKAN/CKAN/blob/1b1d181ec9d92f952834e571f5be3db6cb4ba0d4/AutoUpdate/CKAN-autoupdate.csproj#L45